packages/react-components/src/organisms/ResearchOutputFormSharingCard.tsx

```diff
@@ -203,14 +203,13 @@ const ResearchOutputFormSharingCard: React.FC<
           placeholder="Select subtype"
         />
       )}
-      <LabeledTextArea
+      <LabeledTextEditor
         title="Description"
         subtitle="(required)"
         tip="Add an abstract or a summary that describes this work."
         onChange={onChangeDescription}
         getValidationMessage={() => 'Please enter a description'}
         required
-        enabled={!isSaving}
```

Comment:
maybe we can use this 

```
editor.setEditable(true/false)
```

to set to disabled when is saving (https://lexical.dev/docs/concepts/read-only)